lib/Doctrine/ORM/Query/SqlWalker.php

```diff
@@ -1725,26 +1726,54 @@ public function walkGroupByClause($groupByClause)
         return ' GROUP BY ' . implode(', ', $sqlParts);
     }
 
+    public function walkGroupByPathExpression(Ast\PathExpression $pathExpr): string
```

Comment:
This should maybe be private? Since its an implementation detail of SqlWalker. Otherwise we need a strategy to put it on the interface and other walkers in the future.

Reply:
Either way, I wouldn't put this on the interface. Have you seen #9551?

Reply:
Yes, my bad, this should be of course be private